src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div
+        class = "calendar calendar--start-day--sun calendar--month-length--29"
+      >
+        <div class="day day--1"></div>
+        <div class="day day--2"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
```
it's element, not a block
+ use nth child instead of specifying number directly